tests/negtelnetserver.py

```diff
@@ -23,7 +22,7 @@
 
 # The strings that indicate the test framework is checking our aliveness
 VERIFIED_REQ = b"verifiedserver"
-VERIFIED_RSP = b"WE ROOLZ: {pid}"
+VERIFIED_RSP = "WE ROOLZ: {pid}"
```

Comment:
I don't know Python, but you removed the 'b' on one of these lines and kept it on the other. Was that on purpose?

Reply:
The response is encoded into bytes instead of being a raw byte string below, so this is probably fine.

Reply:
My initial reasoning matches that of @cmeister2.  However the asymmetry has a slight "code smell" of something amiss - e.g. if the server runs on an EBCDIC machine.  But I'm not familiar enough with the test suite to know how character encoding is handled at the test client end.